### ISSUE TITLE
test more preprocessing/scaling methods

### ISSUE THREAD

**Issue by JannisHoch** [JannisHoch/conflict_model#14]
have a look at [this website](https://towardsdatascience.com/scale-standardize-or-normalize-with-scikit-learn-6ccc7d176a02) and implement other preprocessing steps [from skl](https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.MinMaxScaler.html) if results improve
